app/jobs/refresh_github_repos_job.rb

```diff
@@ -10,14 +10,15 @@ def perform(course_id)
       org_repos = github_machine_user.organization_repositories(course.course_organization)
       students = course.roster_students
       summary = ""
-      if org_repos.size == 0
+      if org_repos.respond_to? :each
         summary = "Failed to fetch organization's repos. Either none exist or call to GitHub API failed."
       else
         num_created = 0
         org_repos.each do |github_repo|
           num_created += create_repo_record(github_repo, course, students)
         end
         summary = num_created.to_s + " repos created, " + (org_repos.size - num_created).to_s + " repos refreshed."
+        # RepoCollaboratorsJob.perform_async(course_id)
```

Comment:
any reason why we're leaving commented code here?

Reply:
From one of my above comments:
>The RepoCollaborator job is now a separate job again but is no longer triggered automatically by RefreshGithubReposJob.

It may at some point be desired functionality to update all collaborators from this job as a kind of hard refresh of all GitHub data. For now, we aren't doing that but I left that line commented just as a placeholder if we ever do. Let me know if I should take it out.

Reply:
No that's fine then; just needed to remember why we want to leave it there. Lgtm.